assets/js/googlesitekit/widgets/register-defaults.js

```diff
@@ -102,6 +102,12 @@ export function registerDefaults( widgetsAPI ) {
 			style: WIDGET_AREA_STYLES.BOXES,
 			priority: 1,
 			CTA: ChangeMetricsLink,
+			filterActiveWidgets( select, areaWidgets ) {
+				// Prevent showing only one widget in this area when
+				// only Search Console is shared.
+				// See: https://github.com/google/site-kit-wp/issues/7435
+				return areaWidgets.length === 1 ? [] : areaWidgets;
```

Comment:
Hey @hussain-t, I've noticed this line actually introduces a bug - there is a legitimate case where `areaWidgets` can contain just one item, which is when the widget is the `keyMetricsSetupCTA` widget.

So, this change is currently preventing the Key Metrics Setup CTA widget from being displayed when it should be... We need to bring this back into execution and fix this in a followup PR.

It could be the fix is simply to add an additional check to the condition to ensure that `areaWidgets[0].slug !== 'keyMetricsSetupCTA'`... Maybe there's a slightly better approach, but that should do the trick if not.